runelite-client/src/main/java/net/runelite/client/plugins/gamescaling/GameScalingPlugin.java

```diff
@@ -36,18 +36,18 @@
 import net.runelite.client.plugins.PluginDescriptor;
 
 @PluginDescriptor(
-	name = "Stretched Fixed Mode",
-	description = "Resize the game while in fixed mode",
-	tags = {"resize"},
+	name = "Game Scaling",
```

Comment:
I would probably name this canvas scaling, seems more appropriate

Reply:
I think 'canvas' is unknown to players so I picked 'game' instead. 
What do others who are reading this think?

Reply:
Imo it fits more because canvas usually refers to area where something is drawn, and scaling the drawing area describes what this plugin does perfectly.

Reply:
While "Canvas Scaling" is obviously more accurate, I think many people wouldn't easily recognize its function. I agree with Lotto, "Game Scaling" seems much easier for most people to understand.

Reply:
Wouldn't `Interface Scaling` be more appropriate?

Reply:
If it only scaled resizable, ``Interface Scaling`` would be a good name, since that's what it visibly does in resizable mode. In fixed mode though, it visibly stretches out the whole canvas, which I wouldn't call interface scaling.

Reply:
Imo `Game Scaling` sounds the best and reflects what is really happening , since not only the interface is rescaled but also the game models.

Reply:
I guess we have winner then, I disagree with Interface Scaling too